packages/react/import-component/babel.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const deprecate = require('depd')('hops-react');
+
+module.exports = ({ types: t }) => ({
+  visitor: {
+    ImportDeclaration(path) {
+      const modules = ['hops-react', 'untool', this.opts.module];
+      const source = path.node.source.value;
+      if (!modules.includes(source)) return;
+
+      const specifiers = path.get('specifiers');
+      const specifier = specifiers.find(
+        (specifier) => specifier.node.imported.name === 'importComponent'
+      );
+      if (!specifier) return;
+
+      const bindingName = specifier.node.local.name;
+      const binding = path.scope.getBinding(bindingName);
+
+      binding.referencePaths.forEach((refPath) => {
+        const call = refPath.parentPath;
+        t.assertCallExpression(call);
+
+        const argument = call.get('arguments.0');
+        if (!argument) {
+          throw new Error(
+            '"importComponent" must be called with at least one parameter!'
+          );
+        }
+
+        let importedComponent;
+
+        if (t.isStringLiteral(argument)) {
+          importedComponent = argument.node.value;
+          deprecate(
+            '[DEP001] Using a string as loader for `importComponent` is deprecated and will be removed in a future major release (https://github.com/untool/untool/blob/master/DEPRECATIONS.md).'
```

Comment:
`DEP001` (actually `DEP0001`…) is already reserved for the `hops-config` preset.

Reply:
Also the URL to the `DEPRECATIONS.md` is wrong.

Reply:
See https://github.com/xing/hops/pull/1164#discussion_r412828641